r/tools/nixlibs.R

```diff
@@ -40,17 +43,14 @@ try_download <- function(from_url, to_file) {
 # For local debugging, set ARROW_R_DEV=TRUE to make this script print more
 quietly <- !env_is("ARROW_R_DEV", "true")
 
-# Default is build from source, not download a binary
+# Set LIBARROW_BUILD=FALSE to ensure that we use a previously built libarrow
+# and don't fall back to a full source build
```

Comment:
This is a great comment, should we also keep in here that the default is to fallback here too?

```suggestion
# The default will build from source as a fallback if a binary is not found or shouldn't be used
# Set LIBARROW_BUILD=FALSE to ensure that we use a previously built libarrow
# and don't fall back to a full source build
```